README.md

```diff
@@ -71,6 +71,18 @@ NEXT_IMAGE_DOMAIN=https://va-gov-cms.ddev.site
 
 Now you can run `yarn dev` and data will be coming from your local CMS environment instead.
 
+## Generating the static site
+
+To generate the static pages for https://va.gov, run `yarn export`. This command will generate static pages for all paths that next-build is aware of.
+
+To use the redis cache during your static build, run the following command before running `yarn export`:
```

Comment:
Would it make sense to just always incorporate redis into export and build commands?

Reply:
i'm chewing on this still. I think it might be better to surface enable it via env var or command line through the options handling mechanism

Reply:
I guess put another way...leaving aside ISR scenarios for the moment, and understanding that we don't want redis running during `yarn dev`, could we sandwich `yarn export/build`'s internal commands with

1. stand up redis
2. run the actual export/build stuff
3. tear down redis

And have a flag to disable redis for when we actually want that for a build.

I'd be super happy to have that be a follow-on ticket also. This is just 'what if...' kind of thinking.

Reply:
yeah, i think that sounds reasonable. would be a good follow-up